content/04.results.md

```diff
@@ -30,149 +30,200 @@ Taken together, the results suggest that while bioRxiv has experienced dramatic
+
+### Preprints with similar document embeddings share publication venues
+
+We hypothesized that document embedding might capture information associated with the eventual publication venue.
+We used the annotated preprint-publication pairs as a fully held out test set.
+We embedded all PMC papers except those annotated to have a corresponding preprint into the bioRxiv embedding space.
+For journals, we averaged the document representation of all papers in the journal to produce a journal centroid.
+We used a k-nearest neighbor classifier trained on the remaining PMC documents to identify the most similar published papers and assessed performance - the extent to which the journal a paper appeared in the returned list of ten neighbors - through cross validation and the held out test set (Supplemental Figure {@fig:knn_auc}).
+We found an enrichment of papers published within the same journal among the neighbors of the target paper.
+
+We developed an online app that returns the nearest published papers and journals that are nearest to a preprint.
+Users supply DOIs from _bioRxiv_ or _medRxiv_. <!-- we need to be consistent in how these are represented - should they always be italicized? it feels like a journal name would be - perhaps see what the bioRxiv folks did in their own paper. -->
+The application downloads the article from the preprint repository, converts the PDF to text, calculates a document embedding score, and returns the ten papers and journals with the most similar representations in the embedding space (Figure {@fig:journal_rec_workflow}).
+We also sought to display the position of each preprint in the overall landscape.
+We required an embedding that could be rapidly calculated for new documents, so we used the Sparse Autoencoder for Unsupervised Clustering, Imputation, and Embedding (SAUCIE) approach previously described for the analysis of single-cell gene expression data [@doi:10.1038/s41592-019-0576-7].
+The user-requested preprint's location in this space is then displayed.
+Users can select regions to identify the terms most associated with those regions.
+Users can also explore the terms associated with the top 50 PCs derived from the document embeddings and those PCs vary across the document landscape.
 
 ![
-Here is the workflow of the journal recommender web-app.
-Starting with the homescreen users can paste in a _bioRxiv_ or _medRxiv_ DOI, which sends a request to biorxiv or medrxiv (A).
-Next our app preprocesses the preprint and returns a listing of the top ten most similar papers (B) and the top ten closest journals to the query (C).
-Following the listing, our app manually plots the preprint query onto the Pubmed Central Landscape (D).
-Lastly, users can click on a square within the landscape, which will show bin statistics as well as associated word-odd ratios (E).
+The journal recommender app workflow allows users to examine where an individual preprint falls in the overall document landscape.

```

Comment:
I thought we were shying away from calling it a "recommender" app and moving towards calling it a similarity search app.
```suggestion
The journal similarity search app workflow allows users to examine where an individual preprint falls in the overall document landscape.
```